project_name/settings.py-tpl

```diff
@@ -79,6 +79,10 @@ DATABASES = {
     "default": django_mongodb_backend.parse_uri("mongodb://localhost:27017/{{ project_name }}"),
 }
 
+# Custom database router
```

Comment:
I think we can just write "Database routers", omitting "Custom"
For other comments there is always a blank line between the comment and the setting.